types/google-apps-script/index.d.ts

```diff
@@ -15,20 +19,58 @@
+/// <reference path="google-apps-script.sites.d.ts"/>
+/// <reference path="google-apps-script.slides.d.ts"/>
+/// <reference path="google-apps-script.spreadsheet.d.ts"/>
+
+// Script Services (Other *App / *Service)
 /// <reference path="google-apps-script.html.d.ts"/>
 /// <reference path="google-apps-script.jdbc.d.ts"/>
 /// <reference path="google-apps-script.language.d.ts"/>
 /// <reference path="google-apps-script.lock.d.ts"/>
-/// <reference path="google-apps-script.mail.d.ts"/>
-/// <reference path="google-apps-script.maps.d.ts"/>
-/// <reference path="google-apps-script.optimization.d.ts"/>
 /// <reference path="google-apps-script.properties.d.ts"/>
+/// <reference path="google-apps-script.optimization.d.ts"/>
 /// <reference path="google-apps-script.script.d.ts"/>
-/// <reference path="google-apps-script.sites.d.ts"/>
-/// <reference path="google-apps-script.slides.d.ts"/>
-/// <reference path="google-apps-script.spreadsheet.d.ts"/>
-/// <reference path="google-apps-script.types.d.ts"/>
 /// <reference path="google-apps-script.ui.d.ts"/>
 /// <reference path="google-apps-script.url-fetch.d.ts"/>
 /// <reference path="google-apps-script.utilities.d.ts"/>
 /// <reference path="google-apps-script.xml-service.d.ts"/>
+
+// API Types (Advanced Google Services)
+/// <reference path="apis/adsense/v1.4.d.ts"/>
+/// <reference path="apis/admin/directory_v1.d.ts"/>
+/// <reference path="apis/admin/reports_v1.d.ts"/>
+/// <reference path="apis/bigquery/v2.d.ts"/>
+/// <reference path="apis/calendar/v3.d.ts"/>
+/// <reference path="apis/content/v2.d.ts"/>
+/// <reference path="apis/dfareporting/v2.8.d.ts"/>
+/// <reference path="apis/dfareporting/v3.0.d.ts"/>
+/// <reference path="apis/dfareporting/v3.1.d.ts"/>
+/// <reference path="apis/dfareporting/v3.2.d.ts"/>
+/// <reference path="apis/drive/v2.d.ts"/>
+/// <reference path="apis/appsactivity/v1.d.ts"/>
+/// <reference path="apis/reseller/v1.d.ts"/>
+/// <reference path="apis/licensing/v1.d.ts"/>
+/// <reference path="apis/fusiontables/v1.d.ts"/>
+/// <reference path="apis/fusiontables/v2.d.ts"/>
+/// <reference path="apis/gmail/v1.d.ts"/>
+/// <reference path="apis/analytics/v3.d.ts"/>
+/// <reference path="apis/analyticsreporting/v4.d.ts"/>
+/// <reference path="apis/classroom/v1.d.ts"/>
+/// <reference path="apis/mirror/v1.d.ts"/>
+/// <reference path="apis/sheets/v4.d.ts"/>
+/// <reference path="apis/slides/v1.d.ts"/>
+/// <reference path="apis/plus/v1.d.ts"/>
+/// <reference path="apis/plusDomains/v1.d.ts"/>
+/// <reference path="apis/groupsmigration/v1.d.ts"/>
+/// <reference path="apis/groupssettings/v1.d.ts"/>
+/// <reference path="apis/tagmanager/v1.d.ts"/>
+/// <reference path="apis/tasks/v1.d.ts"/>
+/// <reference path="apis/urlshortener/v1.d.ts"/>
+/// <reference path="apis/youtubeAnalytics/v1.d.ts"/>
+/// <reference path="apis/youtube/v3.d.ts"/>
+/// <reference path="apis/youtubePartner/v1.d.ts"/>
+
+declare module "google-apps-script";
```

Comment:
Are you sure this is what you intended? This creates a new module with `any`-typed contents. Users can import arbitrary names from it with no compile errors.